src/main/java/uk/gov/justice/hmpps/prison/api/resource/SmokeTestHelperResource.java

```diff
@@ -41,5 +43,23 @@ public void imprisonmentDataSetup(@PathVariable("offenderNo") @ApiParam(value =
         service.imprisonmentDataSetup(offenderNo);
     }
 
+    @ApiResponses({
+        @ApiResponse(code = 403, message = "Requires role ROLE_SMOKE_TEST", response = ErrorResponse.class),
+        @ApiResponse(code = 404, message = "Requested resource not found.", response = ErrorResponse.class)})
+    @ApiOperation(value = "Releases this offender, with smoke test data")
+    @PutMapping("/offenders/{offenderNo}/release")
+    @ProxyUser
+    public void releasePrisoner(@PathVariable("offenderNo") @ApiParam(value = "offenderNo", required = true, example = "A1234AA") @NotNull final String offenderNo) {
+        service.releasePrisoner(offenderNo);
+    }
 
+    @ApiResponses({
+        @ApiResponse(code = 403, message = "Requires role ROLE_SMOKE_TEST", response = ErrorResponse.class),
+        @ApiResponse(code = 404, message = "Requested resource not found.", response = ErrorResponse.class)})
+    @ApiOperation(value = "Recalls this offender, with smoke test data")
+    @PutMapping("/offenders/{offenderNo}/recall")
+    @ProxyUser
+    public void recallPrisoner(@PathVariable("offenderNo") @ApiParam(value = "offenderNo", required = true, example = "A1234AA") @NotNull final String offenderNo) {
+        service.recallPrisoner(offenderNo);
+    }
```

Comment:
Am currently using this to ensure the test data is returned to its initial state - additional tests will be added